tools/validator-client-scripts/README.md

```diff
@@ -0,0 +1,32 @@
+# NYM - CLI TOOL
+## Overview
+Set of simple CLI commands to perform basic actions, such as upload contract, bond mixnode, etc.
+Note: to work it requires rust 1.60+ 
+
+## The binary
+    - Build the binary 
+    - cargo build --release 
```

Comment:
you can add you can use `cargo build --release --bin validator-client-scripts` so that it would only build this binary rather than everything in the repo (which should speed things up)

Reply:
fixed in pending inbound commit